.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -0,0 +1,147 @@
+pr: none
+trigger: none
+
+stages:
+  - stage: creating_aks_cluster
+    displayName: "Create AKS Cluster with Cilium"
+    jobs:
+      - job: create_aks_cluster_with_cilium
+        steps:
+          - script: |
+              set -ex        
+              echo "installing kubectl"
+              curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+              sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+              echo "installing helm"
+              curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+              chmod 700 get_helm.sh
+              ./get_helm.sh
+            name: "setup"
+            displayName: "Setup Kubectl and helm"
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                az group create --location $(LOCATION) --name $(RESOURCE_GROUP) --subscription $(SUBSCRIPTION_ID)
+                az account set -s $(SUBSCRIPTION_ID)
+                # Create a vnet
+                az network vnet create -g $(RESOURCE_GROUP) -n overlayvnet --address-prefixes 10.0.0.0/8 --output none
+                subnet=$(az network vnet subnet create -g $(RESOURCE_GROUP) --vnet-name overlayvnet --name nodenet --address-prefix 10.10.0.0/16 -o none --query '{id: id}' --output tsv)
+                # create AKS Cluster
+                az aks create -n $(RESOURCE_GROUP) -g $(RESOURCE_GROUP) -l $(LOCATION) --node-count 10 --network-plugin azure --network-plugin-mode overlay --pod-cidr 192.168.0.0/16 --vnet-subnet-id $subnet -s Standard_D8_v3 --enable-cilium-dataplane --no-ssh-key --yes
+                az aks get-credentials -n $(RESOURCE_GROUP) -g $(RESOURCE_GROUP)

```

Comment:
can you replace these infra scripts with usage of the [hack Makefile](https://github.com/Azure/azure-container-networking/blob/master/hack/swift/Makefile)? This helps us centralize and standardize how clusters are created and reduces maintenance burden when there are changes such as the flag `--enable-cilium-dataplane` -> `--network-dataplane=cilium`

Reply:
ref https://github.com/Azure/azure-container-networking/blob/master/.pipelines/singletenancy/cilium/cilium-e2e-step-template.yaml, this pipeline should look very similar to that one

Reply:
Sure, let me update it.